packages/ai-ide/src/browser/ide-chat-welcome-message-provider.tsx

```diff
@@ -57,30 +60,63 @@ const TheiaIdeAiLogo = ({ width = 200, height = 200, className = '' }) =>
 @injectable()
 export class IdeChatWelcomeMessageProvider implements ChatWelcomeMessageProvider {
 
+    /**
+     * @deprecated not needed by this class anymore
+     */
```

Comment:
We should remove this field if we don't need it anymore. The same for the deprecated method.